cputree.c

```diff
@@ -45,12 +45,11 @@ GList *cache_domains;
 GList *packages;
 
 int cache_domain_count;
-int core_count;
 
 /* Users want to be able to keep interrupts away from some cpus; store these in a cpumask_t */
 cpumask_t banned_cpus;
 
-cpumask_t cpu_possible_map;
+cpumask_t cpu_online_map;
```

Comment:
if you're going to do this, you may as well remove the cpu_possible_map extern and its associated macros.

Reply:
cpu_present_map is also unused too, is it need to remove together ?
thanks.

Reply:
may as well, sure

Reply:
ok, patch has been updated now.

Reply:
looks good